dbt_style_guide.md

```diff
@@ -103,19 +164,71 @@ select * from filtered_events
 ## SQL style guide
 
 - Use trailing commas
-- Indents should be four spaces (except for predicates, which should line up with the `where` keyword)
-- Lines of SQL should be no longer than 80 characters
+
+- Indents should be two spaces (except for predicates, which should line up with the `where` keyword)

```

Comment:
👀 i think my editor defaults to 4 -- thoughts?

Reply:
same... I prefer to be able to just hit tab

Reply:
From a purely pragmatic perspective, there may be a _lot_ a momentum for 4 spaces w/in our codebase already.
But, I'm also biased -- 4 spaces 4 lyfe.

Reply:
👍  I’m down for the default, I always found that a little annoying to deal with! In our old version it had 2 and 4 listed in different sections 🤔

Reply:
Done! [Commit](https://github.com/dbt-labs/corp/pull/63/commits/b5d1dc06dfcac12287e7e844b79ce051ea3d6e00)